EpsilonAuxBmsV2/Tests/tests/OrionInterfaceTest/OrionInterfaceTest.c

```diff
@@ -17,241 +12,75 @@ osThreadId_t commonContactorGatekeeperTaskHandle;
-    expectedAuxStatus.strobeBmsLight = 0;
-    expectedAuxStatus.allowCharge = 1;
-    expectedAuxStatus.allowDischarge = 1;
-    expectedAuxStatus.orionCanReceivedRecently = 1;
-    expectedAuxStatus.chargeShouldTrip = 0;
-    expectedAuxStatus.dischargeShouldTrip = 0;
-
-    orionInterface(&message);
-    assertAuxStatusEqual(expectedAuxStatus);
-}
-
-// Orion message hasn't been received recently via CAN
-// Test that aux status is set appropriately
-void test_NoOrionCanMessageReceivedRecently()
-{
-    setOrionInterfaceOsExpects(0);
-    setContactorState(CLOSED, CLOSED, 1);
-    setContactorEventFlagsExpects(0);
-
-    GPIO_PinState expectedOrionDischargeSense = GPIO_PIN_SET;
-    GPIO_PinState expectedOrionChargeSense = GPIO_PIN_SET;
-
-    setOrionEnableSenseExpects(expectedOrionDischargeSense, expectedOrionChargeSense);
-
-    // Explictly define what we are expecting
-    setNominalAuxStatus();
-    AuxStatus expectedAuxStatus = auxStatus;
-    expectedAuxStatus.orionCanReceivedRecently = 0;
-
-    orionInterface(&message);
-    assertAuxStatusEqual(expectedAuxStatus);
-}
-
-// Orion requests to disconnect discharge contactor
-// Test that the discharge contactor off event is triggered & statuses are correct
-void test_NoDischargeSenseShouldTurnOffPinAndUpdateStatuses()
-{
-    setOrionInterfaceOsExpects(1);
-    setTripExpects(0, 0, 0);
-    setContactorState(CLOSED, CLOSED, 1);
-    setContactorEventFlagsExpects(DISCHARGE_OFF);
-    osThreadSetPriority_ExpectAndReturn(dischargeContactorGatekeeperTaskHandle, osPriorityRealtime, 0);
-
-    GPIO_PinState expectedOrionDischargeSense = GPIO_PIN_RESET;
-    GPIO_PinState expectedOrionChargeSense = GPIO_PIN_SET;
-    setOrionEnableSenseExpects(expectedOrionDischargeSense, expectedOrionChargeSense);
-
-    setNominalAuxStatus();
-    AuxStatus expectedAuxStatus = auxStatus;
-    expectedAuxStatus.allowDischarge = 0;
-
-    orionInterface(&message);
-
-    assertAuxStatusEqual(expectedAuxStatus);
-}
-
-// Orion requests to disconnect charge contactor
-// Test that the charge contactor off event is triggered & statuses are correct
-void test_NoChargeSenseShouldTurnOffPinAndUpdateStatuses()
+void runOrionInterfaceTests()
 {
-    setOrionInterfaceOsExpects(1);
-    setTripExpects(0, 0, 0);
-    setContactorState(CLOSED, CLOSED, 1);
-    setContactorEventFlagsExpects(CHARGE_OFF);
-    osThreadSetPriority_ExpectAndReturn(chargeContactorGatekeeperTaskHandle, osPriorityRealtime, 0);
-
-    GPIO_PinState expectedOrionDischargeSense = GPIO_PIN_SET;
-    GPIO_PinState expectedOrionChargeSense = GPIO_PIN_RESET;
-    setOrionEnableSenseExpects(expectedOrionDischargeSense, expectedOrionChargeSense);
-
-
-    setNominalAuxStatus();
-    AuxStatus expectedAuxStatus = auxStatus;
-    expectedAuxStatus.allowCharge = 0;
-
-    orionInterface(&message);
-
-    assertAuxStatusEqual(expectedAuxStatus);
+    // RUN_TEST(test_onlyUpdateAllowCharge);
+    // RUN_TEST(test_onlyUpdateAllowDischarge);
```

Comment:
Is there a reason we commented out these tests?

Reply:
They should be running.